prover/protocol/compiler/vortex/compiler.go

```diff
@@ -86,6 +86,7 @@ func Compile(blowUpFactor int, options ...VortexOp) func(*wizard.CompiledIOP) {
 		// Registers the prover and verifier steps
 		comp.SubProvers.AppendToInner(lastRound+1, ctx.ComputeLinearComb)
 		comp.SubProvers.AppendToInner(lastRound+2, ctx.OpenSelectedColumns)
+		comp.InsertVerifier(lastRound, ctx.explicitPublicEvaluation, ctx.gnarkExplicitPublicEvaluation)
```

Comment:
This is separated from GnarkVerify because we want to recurse this verifier step but not gnark verify which is already handled by the self-recursion mechanism.